src/org/thoughtcrime/securesms/preferences/AdvancedPreferenceFragment.java

```diff
@@ -76,16 +78,28 @@ public void onCreate(Bundle paramBundle) {
     bccSelfCheckbox.setOnPreferenceChangeListener((preference, newValue) -> {
       boolean enabled = (Boolean) newValue;
       dcContext.setConfigInt(CONFIG_BCC_SELF, enabled? 1 : 0);
+      DcHelper.getAccounts(getContext()).startIo();
```

Comment:
why is io not stopped before here?

Reply:
Doesn't seem needed, idk why I added it

Reply:
i just had a rough look at core, it seems not to be needed.